pkgs/tools/wayland/sirula/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, gtk3
+, gtk-layer-shell
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "sirula";
+  version = "574725bc307fc704c42380cd0fa50b0b80c4764d";
```

Comment:
Version should be the date with a `unstable-` added before it (https://nixos.org/manual/nixpkgs/stable/#sec-package-naming)
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".